main.go

```diff
@@ -73,6 +75,10 @@ func init() {
 }
 
 func main() {
+	const (
+		tokenCacheDefaultMaxSize = 0
```

Comment:
Let's enable cacheing by default here and in all other controllers. The cache implementation has been shipped for some time now in Flux Operator and no issues were found in production envs. 

```suggestion
		tokenCacheDefaultMaxSize = 100
```

Reply:
Fixed, thanks!